### DISCUSSION TITLE
Can I use cachelib as persist cache like bcache

### DISCUSSION THREAD

**Discussion by daxiguafan** [facebook/CacheLib#111] [Q&A] [answered]
Can I use cachelib as a hard disk cache? So when I successfully write data to cachelib, if the machine is powered off and back on at this time, I hope the data will not be lost, just like bcache, until the item is evicted (will be saved to hard disk).

**Comment by leozzx** [comment, marked as answer]
see the discussion in #113, cachelib does not offer crash consistent storage of items. Durability is usually not a concern for a cache.
